packages/components/space/src/useSpace.ts

```diff
@@ -116,21 +137,26 @@ export function useSpace(props: ExtractPropTypes<typeof defaultProps>) {
   )
 
   const containerStyle = computed(() => {
-    const wrapKls: CSSProperties =
-      props.wrap || props.fill
-        ? { flexWrap: 'wrap', marginBottom: `-${verticalSize.value}px` }
-        : null
+    const flexGapStyles: CSSProperties = flexGapSupported()
+      ? { gap:`${verticalSize.value}px ${horizontalSize.value}px` }
+      : { marginBottom: `-${verticalSize.value}px` }
+
+    const wrapKls: CSSProperties = props.wrap || props.fill
+      ? { ...flexGapStyles , flexWrap: 'wrap' }
+      : flexGapSupported() ? { gap:`${verticalSize.value || horizontalSize.value}px` } : null
+
```

Comment:
Could we cache the result? So that we can avoid the overhead of constantly creating and inserting and remove DOM element?

Even though computed does cache the result. But I think we could cache the result since the environment detecting is a one time thing.

Plus, you should avoid using raw DOM APIs without detecting the current environment. E.g. SSR context.

Reply:
BTW, the code seems legit to me.

Reply:
很抱歉提出了一个考虑不那么成熟的代码，我会在时间允许的情况下去修复它的这些问题。

Reply:
It is ok. This is why code review is so important. If you need any help on contributing, please feel free to ping me in any thread. 

BTW, if you need to detect whether you are in `browser` you should try

```
import isServer from '@element-plus/utils/isServer`
```